frontend/public/locales/zh/monitoring.json

```diff
@@ -0,0 +1,38 @@
+{
+  "View in Metrics": "在指标中查看",
+  "Name": "名称",
+  "Firing alerts": "发出警报",
+  "State": "状态",
+  "Creator": "创建者",
+  "Alerts": "警报",
+  "Alert details": "警报详细信息",
+  "Severity": "严重性",
+  "Description": "描述",
+  "Summary": "概述",
+  "Message": "消息",
+  "Source": "Source",
```

Comment:
"Source" could be translated.

Reply:
This was not addressed.

Reply:
I can see that this had been addressed in the translation files, will send the files over again just in case.

Reply:
This is all set.